lib/utils/compose_ts.ts

```diff
@@ -43,6 +43,7 @@ import {
 import type { DeviceInfo } from './device/api';
 import { getBalenaSdk, getChalk, stripIndent } from './lazy';
 import Logger = require('./logger');
+import * as yaml from 'js-yaml';
```

Comment:
I think we generally prefer to load modules at the point where they are used, no? So importing inside the `mergeDevComposeOverlay` function.